menas/src/test/scala/za/co/absa/enceladus/service/DatasetServiceTest.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018-${today.year} ABSA Group Limited
```

Comment:
I guess the intelligence copyright template didn't quite work out here

Reply:
Resolved